source/sass/glyphs.scss

```diff
@@ -1,6 +1,3 @@
-$default-glyph-path: "../_images/glyphs";
```

Comment:
Moved to [source/sass/variables.scss](https://github.com/mozilla/foundation.mozilla.org/pull/6528/files#diff-d2049013185550a0f0a347ec91292d0f766e9bdd8dbdd9a94769d5fa8673723d)